### PR TITLE
Including PMCID in chemical database ID

### PR DESCRIPTION
**Pull request by DeniseSl22**
https://github.com/enanomapper/ontologies/issues/300

@egonw ; please also check my comments in the issue, regarding adding a DOI, and maybe adding a subclass "literature database identifier" .

### CODE REVIEW

File: internal-dev/cheminf-ext.owl [hunk 1]
```diff
@@ -82,7 +74,13 @@
         <rdfs:label xml:lang="en">cdk descriptors</rdfs:label>
     </owl:Class>
     
+    <!-- http://purl.enanomapper.org/onto/ENM_1200002 -->
 
+    <owl:Class rdf:about="http://purl.enanomapper.org/onto/ENM_1200002">
+        <rdfs:subClassOf rdf:resource="http://semanticscience.org/resource/CHEMINF_000464"/>
+        <rdfs:isDefinedBy>PMCIDr is a unique identifier for publications archived in PubMed Central, which may also be indexed in PubMed. </rdfs:isDefinedBy>
```
Comment (egonw) [inline internal-dev/cheminf-ext.owl:81]: The `r` after `PMCID` in the description is not intented, I guess?

**Review by egonw — CHANGES_REQUESTED** [review]
Better is even to first add the PMCID to the OBI ontology:

https://raw.githubusercontent.com/obi-ontology/obi/master/src/ontology/obi-edit.owl

Example in OBI (PubMed ID): 

image

File: internal-dev/cheminf-ext.owl (hunk 1 above)
Comment (DeniseSl22) [reply internal-dev/cheminf-ext.owl:81]: Sorry didn't see that, must be a typo. I'll fix it right away!

**Review by egonw — APPROVED** [review]

**Comment by egonw** [comment]
Merged in manually bc of merge conflict

**Pull request closed without merging**